### PR TITLE
Replace Image + PDF transform with OCR transform

### PR DESCRIPTION
**Pull request by DhruvaBansal00**

### CODE REVIEW

File: src/autolabel/transforms/ocr.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+class OCRTransform(BaseTransform):
+    """This class is used to extract text from any document using OCR. The output columns dictionary for this class should include the keys 'content_column' and 'metadata_column'
+
+    This transform supports the following image formats: PDF, PNG, JPEG, TIFF, JPEG 2000, GIF, WebP, BMP, and PNM
+    """
+
+    COLUMN_NAMES = [
+        "content_column",
+        "metadata_column",
```
Comment (nihit) [inline src/autolabel/transforms/ocr.py:16]: we can remove this column - it just needs to return the extracted text

File: src/autolabel/transforms/ocr.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    ]
+
+    def __init__(
+        self,
+        cache: BaseCache,
+        output_columns: Dict[str, Any],
+        file_path_column: str,
```
Comment (nihit) [inline src/autolabel/transforms/ocr.py:23]: simplify this to "url_column" or "input_column" - we don't need to support any local filepaths, just http/https urls. We'll download it, and if we cannot then return a `TransformError`

File: src/autolabel/transforms/ocr.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        lang: str = None,
+    ) -> None:
+        super().__init__(cache, output_columns)
+        self.file_path_column = file_path_column
+        self.lang = lang
+
+        try:
+            from PIL import Image
+            import pytesseract
+            from pdf2image import convert_from_path
```
[the remark this answers is not in the document]
Comment (DhruvaBansal00) [reply src/autolabel/transforms/ocr.py:32]: pdf2image is already a part of the minimal autolabel requirements

[the remark this answers is not in the document]
Comment (DhruvaBansal00) [reply src/autolabel/transforms/ocr.py:32]: We do however need to add a line to install poppler-utils in all our docker images now - adding them right now

**Review by nihit — APPROVED** [review]

**Pull request merged**